modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/ElixirClientCodegen.java

```diff
@@ -72,114 +71,111 @@ public ElixirClientCodegen() {
-                        ClientModificationFeature.BasePath
-                )
+                        ClientModificationFeature.BasePath)
                 .includeDataTypeFeatures(
-                        DataTypeFeature.AnyType
-                )
-        );
+                        DataTypeFeature.AnyType));
 
         // set the output folder here
         outputFolder = "generated-code/elixir";
 
         /*
-         * Models.  You can write model files using the modelTemplateFiles map.
+         * Models. You can write model files using the modelTemplateFiles map.
          * if you want to create one template for file, you can do so here.
-         * for multiple files for model, just put another entry in the `modelTemplateFiles` with
+         * for multiple files for model, just put another entry in the
+         * `modelTemplateFiles` with
          * a different extension
          */
         modelTemplateFiles.put(
                 "model.mustache", // the template to use
-                ".ex");       // the extension for each file to write
+                ".ex"); // the extension for each file to write
 
         /**
-         * Api classes.  You can write classes for each Api file with the apiTemplateFiles map.
-         * as with models, add multiple entries with different extensions for multiple files per
+         * Api classes. You can write classes for each Api file with the
+         * apiTemplateFiles map.
+         * as with models, add multiple entries with different extensions for multiple
+         * files per
          * class
          */
         apiTemplateFiles.put(
-                "api.mustache",   // the template to use
-                ".ex");       // the extension for each file to write
+                "api.mustache", // the template to use
+                ".ex"); // the extension for each file to write
 
         /**
-         * Template Location.  This is the location which templates will be read from.  The generator
+         * Template Location. This is the location which templates will be read from.
+         * The generator
          * will use the resource stream to attempt to read the templates.
          */
         templateDir = "elixir";
 
         /**
-         * Reserved words.  Override this with reserved words specific to your language
-         * Ref: https://github.com/itsgreggreg/elixir_quick_reference#reserved-words
+         * Reserved words. Override this with reserved words specific to your language
+         * Ref: https://hexdocs.pm/elixir/1.13/syntax-reference.html#reserved-words
          */
         reservedWords = new HashSet<>(
                 Arrays.asList(
-                        "nil",
                         "true",
                         "false",
-                        "__MODULE__",
-                        "__FILE__",
-                        "__DIR__",
-                        "__ENV__",
-                        "__CALLER__")
```

Comment:
Although I'm pretty sure that nobody will ever name his variable like this in a serious openapi specification, this would still lead to an error for the client and should (IMHO) kept as reserved words

Reply:
When I'm thinking about it, this would also be a great chance to add `__struct__`to the list of reserved words

Reply:
Maybe the `_` on its own could be added here too, to resolve the example from #18661?

```
    error: invalid use of _. _ can only be used inside patterns to ignore values and cannot be used in expressions. Make sure you are inside a pattern or change it accordingly
```

(also I'm currently a bit perplexed about the `--name-mappings` not working: `ElixirClientCodegen` *does not* override `toVarName`, and `DefaultCodegen`'s implementation *does* do the `nameMapping.containsKey(name)` thing, and yet they don't apply 0.o)

Reply:
Hi @valpackett,

Apologies for missing your issue; I'm not notified about issues unless mentioned directly.

From what I can see, there isn't currently any variable mapping implemented that would allow a `_` variable in a struct. During the request/response process, an `underline_var` would need to be mapped to `_` in the request and from `_` back to `underline_var` in the response.

Therefore, adding `_` to the reserved list would only prevent you from using such as an operation ID or module and would not resolve your issue.

I suggest moving forward with this PR and addressing your issue in a separate PR, as it seems to be a more comprehensive task.